java/query/query-lambda/src/main/java/sleeper/query/lambda/WarmQueryExecutorLambda.java

```diff
@@ -0,0 +1,156 @@
+ * remain warm for genuine queries.
+ */
+@SuppressWarnings("unused")
+public class WarmQueryExecutorLambda implements RequestHandler<ScheduledEvent, Void> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(WarmQueryExecutorLambda.class);
+
+    private Instant lastUpdateTime;
+    private InstanceProperties instanceProperties;
+    private TablePropertiesProvider tablePropertiesProvider;
+    private final AmazonSQS sqsClient;
+    private final AmazonS3 s3Client;
+    private final AmazonDynamoDB dynamoClient;
+    private QueryMessageHandler messageHandler;
+    private SqsQueryProcessor processor;
+
+    public WarmQueryExecutorLambda() throws ObjectFactoryException {
+        this(AmazonS3ClientBuilder.defaultClient(), AmazonSQSClientBuilder.defaultClient(),
+                AmazonDynamoDBClientBuilder.defaultClient(), System.getenv(CONFIG_BUCKET.toEnvironmentVariable()));
+    }
+
+    public WarmQueryExecutorLambda(AmazonS3 s3Client, AmazonSQS sqsClient, AmazonDynamoDB dynamoClient, String configBucket) throws ObjectFactoryException {
+        this.s3Client = s3Client;
+        this.sqsClient = sqsClient;
+        this.dynamoClient = dynamoClient;
+        instanceProperties = loadInstanceProperties(s3Client, configBucket);
+        tablePropertiesProvider = new TablePropertiesProvider(instanceProperties, s3Client, dynamoClient);
+    }
+
+    @Override
+    public Void handleRequest(ScheduledEvent event, Context context) {
+        LOGGER.info("Starting to build queries for the tables");
+        S3TableProperties.getStore(instanceProperties, s3Client, dynamoClient)
+                .streamAllTables()
+                .forEach(tableProperty -> {
+                    Schema schema = tableProperty.getSchema();
+                    Field field = schema.getRowKeyFields().get(0);
+
+                    // Create the max and min values for the query. We don't care what they are as long as the query runs
+                    Type type = field.getType();
+                    Object min, max;
+                    if (type instanceof IntType) {
+                        min = 0;
+                        max = 1;
+                    } else if (type instanceof LongType) {
+                        min = 0;
+                        max = 1;
+                    } else if (type instanceof StringType) {
+                        min = "a";
+                        max = "aa";
+                    } else if (type instanceof ByteArrayType) {
+                        min = new ByteArrayType();
+                        max = new ByteArrayType();
+                    } else if (type instanceof MapType) {
+                        min = new HashMap<>();
+                        max = new HashMap<>();
+                    } else if (type instanceof ListType) {
+                        min = new ArrayList<>();
+                        max = new ArrayList<>();
```

Comment:
A row key can only be a primitive type right now.

Reply:
Removed, only uses primitivies